src/vs/workbench/services/extensions/common/extensionManifestPropertiesService.ts

```diff
@@ -68,7 +68,7 @@ export class ExtensionManifestPropertiesService extends Disposable implements IE
 			this._configuredExtensionWorkspaceTrustRequestMap.set(id, configuredExtensionWorkspaceTrustRequests[id]);
 		}
 
-		// Workspace trust request type (products.json)
```

Comment:
I think there should be a typo here, because there seems to be no products.json but product.jon file.